backends/platform/android/android.cpp

```diff
@@ -1024,4 +1024,112 @@ void *OSystem_Android::getOpenGLProcAddress(const char *name) const {
 }
 #endif
 
+static const char *helpTabs[] = {
```

Comment:
Not sure, but I think I would suggest:
````
static const char * const helpTabs[] = {
````
This should help compiler in RAM optimisation, by indicating that the underlying character arrays are constant, not just the pointers to them.

Several other cases of this in various embedded help strings.